src/building/dock.h

```diff
@@ -12,6 +16,11 @@ typedef enum {
     SHIP_DOCK_REQUEST_7_ANY = 7,
 } ship_dock_request_type;
 
+typedef struct handled_good {
```

Comment:
Since this is only used on a single C file, move it there. No need to pollute the header.